### PR TITLE
govet: add more analyzers

### PR DESCRIPTION
**Pull request by ernado**
This allows enabling more analyzers.
No defaults were changed.

Command that helped me to produce full list:
```bash
# in golang.org/x/tools
ls go/analysis/passes/ | grep -vP '(README|internal)' | awk '$0=$0".Analyzer,"'
```

### CODE REVIEW

File: pkg/golinters/govet.go [hunk 1]
```diff
@@ -41,18 +49,26 @@ func getAllAnalyzers() []*analysis.Analyzer {
 		atomic.Analyzer,
 		atomicalign.Analyzer,
 		bools.Analyzer,
+		buildssa.Analyzer,
```
Comment (jirfag) [inline pkg/golinters/govet.go]: but this analyzer in internal and exists just for building SSA: it reports no issues

File: pkg/golinters/govet.go [hunk 2]
```diff
@@ -41,18 +49,26 @@ func getAllAnalyzers() []*analysis.Analyzer {
...
 		buildtag.Analyzer,
 		cgocall.Analyzer,
 		composite.Analyzer,
 		copylock.Analyzer,
+		ctrlflow.Analyzer,
```
Comment (jirfag) [inline pkg/golinters/govet.go]: the same

File: pkg/golinters/govet.go [hunk 3]
```diff
@@ -41,18 +49,26 @@ func getAllAnalyzers() []*analysis.Analyzer {
...
+		deepequalerrors.Analyzer,
 		errorsas.Analyzer,
+		findcall.Analyzer,
 		httpresponse.Analyzer,
+		inspect.Analyzer,
```
Comment (jirfag) [inline pkg/golinters/govet.go]: I guess it's internal too

File: pkg/golinters/govet.go (hunk 1 above)
Comment (ernado) [reply pkg/golinters/govet.go]: Can we automatically filter internal analyzers by some condition?

Comment (jirfag) [reply pkg/golinters/govet.go]: I don't know how to do it

Comment (ernado) [reply pkg/golinters/govet.go]: Seems like the following should work:
```go
func isInternal(a *analysis.Analyzer) bool {
	return a.ResultType != nil
}
```

Comment (ernado) [reply pkg/golinters/govet.go]: I've created the #763 to address this issue. 
I still want to import every analyzer and explicitly external ones, seems like it is most maintainable approach.

**Pull request merged**